CONTRIBUTING.md

```diff
@@ -231,17 +231,16 @@ At the moment, only a single major release will be supported, so all point
 releases will be based off of that release.
 
 - start on `vX.XX.X`
-- checkout a new branch `pre-rc/vX.X.X`
+- checkout a new branch `rc/vX.X.X`
 - cherry pick the desired changes from `master`
   - these changes should be small and NON-BREAKING (both API and state machine)
 - add entries to CHANGELOG.md and remove corresponding pending log entries
-- checkout a new branch `rc/vX.X.X` based off of `vX.XX.X`
-- create a PR merging `pre-rc/vX.X.X` into `rc/vX.X.X`
+- checkout a new branch `release/vX.X.X` based off of `vX.XX.X`
+- create a PR merging `rc/vX.X.X` into `release/vX.X.X`
 - run tests and simulations (noted in [Release Procedure](#release-procedure))
-- after tests and simulation have successfully completed, create the release branch `release/vX.XX.X` from the `RC` branch
-- delete the `pre-rc/vX.X.X` and `RC` branches
```

Comment:
We still need to delete all the RC branches.

Reply:
i believe all branches are deleted after merging unless there is another branch depending on it or its protected

Reply:
RC branches have to be manually deleted AFAIK. I could be wrong.